_includes/legacy/sections/android-operating-systems.html

```diff
@@ -1,39 +1,33 @@
+  badges="info:AOSP"
+  website="https://grapheneos.org/"
+  github="https://github.com/GrapheneOS/"
+%}
 
 {% include legacy/cardv2.html
   title="CalyxOS"
   image="/assets/img/legacy_svg/3rd-party/calyxos.svg"
-  description='CalyxOS is a custom ROM based on the Android Open Source Project, with support for Google Pixel devices and the Xiaomi Mi A2. It aims to be secure and private by default, including a number of good choices for default applications. It also supports <a href="https://source.android.com/security/verifiedboot">verified boot</a>.'
+  description='CalyxOS is a custom ROM based on the Android Open Source Project, with support for <a href="https://calyxos.org/docs/guide/device-support/">Google Pixel devices and the Xiaomi Mi A2</a>. It aims to be secure and private by default, and includes a number of good choices for default applications. Some of its features include optional <a href="https://microg.org/">MicroG</a> support and a built-in <a href="https://calyxos.org/docs/tech/datura-details/">firewall</a>. It should be noted that CalyxOS lacks a lot of the security hardening that GrapheneOS has, such as a hardened memory allocator, hardened webview, more <a href="https://grapheneos.org/usage#exec-spawning">secure spawning system</a>, and so on.'
   badges="info:AOSP"
   website="https://calyxos.org"
   gitlab="https://gitlab.com/calyxos"
 %}
 
 {% include legacy/cardv2.html
-  title="GrapheneOS"
-  image="/assets/img/legacy_svg/3rd-party/grapheneos.svg"
-  image-dark="/assets/img/legacy_svg/3rd-party/grapheneos-dark.svg"
-  description='GrapheneOS (formerly known as CopperheadOS) is a free and open-source security- and privacy-focused mobile operating system built on top of the Android Open Source Project. It specifically targets devices offering strong hardware security, currently only supporting Google Pixel phones.'
+  title="DivestOS"
+  image="/assets/img/legacy_png/3rd-party/divestos.png"
+  description='<p>DivestOS is a soft-fork of <a href="https://lineageos.org/">LineageOS</a> with signed builds, automated kernel vulnerability (CVE) patcher, bundled <a href="https://www.f-droid.org/">F-Droid</a> and <a href="https://github.com/microg/UnifiedNlp">UnifedNlp</a>.</p>
+  <p>Not all of its supported devices support <a href="https://source.android.com/security/verifiedboot">verified boot</a>, and DivestOS does not handle firmware updates well currently. For Pixel devices, we still recommend using GrapheneOS or CalyxOS. For other <a href="https://divestos.org/index.php?page=devices&base=LineageOS">supported devices</a>, DivestOS is a good option.</p>'
   badges="info:AOSP"
-  website="https://grapheneos.org/"
-  github="https://github.com/GrapheneOS/"
+  website="https://www.divestos.org/"
+  tor="divestoseb5nncsydt7zzf5hrfg44md4bxqjs5ifcv4t7gt7u6ohjyyd.onion"
```

Comment:
wasn't clickable, apologies
```suggestion
  tor="http://divestoseb5nncsydt7zzf5hrfg44md4bxqjs5ifcv4t7gt7u6ohjyyd.onion/"
```